zerver/lib/types.py

```diff
@@ -189,6 +190,7 @@ class SubscriptionStreamDict(TypedDict):
     desktop_notifications: bool | None
     email_notifications: bool | None
     first_message_id: int | None
+    is_recently_active: int | None
```

Comment:
Wait isn't this a bool? Also you should put it in alphabetical/sorted order.

(I assume this is leftover from a previous `last_message_id` iteration).

Reply:
Oops, changed that.